src/js/debugger/app.js

```diff
@@ -0,0 +1,1883 @@
+/**
+ * Prevents parent element from scrolling when hovering over debugger elements.
+ */
+
```

Comment:
Can any of this file be broken up into more maintainable pieces and then just reassembled into the final js file by Grunt?  Perhaps pieces that are not custom can each get their own file?  I can help if you need.